### PR TITLE
Fix: Doctor login and registration with supabase authentication

### LINKED ISSUE #13: Fix: Supabase authentication

### PR DESCRIPTION
**Pull request by mr3nz1**
### What does this pull request have?
It contains login and registration functionalities that align with the patient's app

### Description
<div style="display: flex; flex-direction: row; justify-content: space-around;">
        Screenshot 286
        Screenshot 285
        Screenshot 287
    </div>

### CODE REVIEW

**Review by Irirwanirira — APPROVED** [review]
Good Job on implementing all these changes

**Review by munezeromicha — APPROVED** [review]
Good Job!

File: src/app/assets/images/Ellipse.png
Comment (Nkbtemmy) [inline src/app/assets/images/Ellipse.png:1]: We agreed on using black doctors !!

File: src/app/components/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                                }`}
+                                onClick={() => (window.location.href = item.path)}
+                            >
+                                <div className="mr-3">{item.icon}</div>
+                                <a className="text-lg">{item.name}</a>
+                            </li>
+                        ))}
+                    </ul>
+                </nav>
+            </aside> */}
```
Comment (Nkbtemmy) [inline src/app/components/layout.tsx]: Remove commented codes !!

File: src/app/dashboard/layout.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+export default function Layout({ children }: Props) {
+  const { isLoggedIn, status, user } = useAuth();
+  const router = useRouter();
+
+  console.log("###################");
+  console.log("isLoggedIn: " + isLoggedIn);
+  console.log("status: " + status);
+  console.log("User: ", user);
+  console.log("###################");
```
Comment (Nkbtemmy) [inline src/app/dashboard/layout.tsx]: Also remove this consoles !!

**Review by Nkbtemmy — CHANGES_REQUESTED** [review]
Remember also to provide .env.example file with variables without values !!

File: src/app/assets/images/Ellipse.png
Comment (Irirwanirira) [reply src/app/assets/images/Ellipse.png:1]: As data are still static we will be removing them while integrating, we have African image in the backend

**Review by Nkbtemmy — APPROVED** [review]

**Pull request merged**